lib/preprocessed/ToString.post.sol

```diff
@@ -60,24 +60,24 @@ function binBranchToString(Bin tick) pure returns (string memory) {
   return string.concat(vm.toString(tick.posInRoot()), "->", vm.toString(tick.posInLevel1()), "[", vm.toString(tick.level1Index()), "]->", vm.toString(tick.posInLevel2()), "[", vm.toString(tick.level2Index()), "]->", vm.toString(tick.posInLevel3()), "[", vm.toString(tick.level3Index()), "]->", vm.toString(tick.posInLeaf()), "[", vm.toString(tick.leafIndex()), "]");
 }
 
-function toString(Bin tick) pure returns (string memory ret) {
+function toString(Bin bin) pure returns (string memory ret) {
   string memory suffix;
-  if (MIN_BIN > Bin.unwrap(tick) || Bin.unwrap(tick) > MAX_BIN) {
+  if (MIN_BIN > Bin.unwrap(bin) || Bin.unwrap(bin) > MAX_BIN) {
     suffix = "out of range";
-  } else if (MIN_BIN_ALLOWED > Bin.unwrap(tick) || Bin.unwrap(tick) > MAX_BIN_ALLOWED) {
-    suffix = "out of tick range";
+  } else if (MIN_BIN_ALLOWED > Bin.unwrap(bin) || Bin.unwrap(bin) > MAX_BIN_ALLOWED) {
+    suffix = "out of bin range";
   } else {
-    suffix = tickToString(TickLib.fromBin(tick,1));
+    suffix = toString(BinLib.toNearestTick(bin,1));
```

Comment:
Shouldn't this use the `bin.method` syntax?
```suggestion
    suffix = toString(bin.toNearestTick(1));
```

Reply:
fixed in 8ef390398905f50d1d11d6462ed559fe86b0a90b